src/borg/archiver.py

```diff
@@ -78,8 +78,10 @@ def wrapper(self, args, **kwargs):
             elif location.proto == 'ssh':
                 repository = RemoteRepository(location, create=create, lock_wait=self.lock_wait, lock=lock, args=args)
```

Comment:
did you check the remote code also?

if it needs a RPC interface change, leave a comment there, but do not do incompatible RPC interface changes.